robotmodels/dingo_kinova/xml/dingo_kinova.xml

```diff
@@ -0,0 +1,89 @@
+                <joint name="arm_joint_3" pos="0 0 0" axis="0 0 1" limited="true" range="-2.61 2.61"/>
+                <geom type="mesh" contype="0" conaffinity="0" group="1" density="0" rgba="0.803 0.824 0.82 1" mesh="forearm_link"/>
+                <geom type="mesh" rgba="0.803 0.824 0.82 1" mesh="forearm_link"/>
+                <body name="arm_lower_wrist_link" pos="0 -0.14 0.02" quat="0.707105 0.707108 0 0">
+                  <inertial pos="0.00575149 0.0100044 0.0871921" quat="0.700403 -0.13124 -0.134345 0.688595" mass="0.526934" diaginertia="0.000785199 0.000756398 0.000275722"/>
+                  <joint name="arm_joint_4" pos="0 0 0" axis="0 0 1" limited="true" range="-2.57 2.57"/>
+                  <geom type="mesh" contype="0" conaffinity="0" group="1" density="0" rgba="0.803 0.824 0.82 1" mesh="lower_wrist_link"/>
+                  <geom type="mesh" rgba="0.803 0.824 0.82 1" mesh="lower_wrist_link"/>
+                  <body name="arm_upper_wrist_link" pos="0.0285 0 0.105" quat="0.707105 0 0.707108 0">
+                    <inertial pos="0.0805652 0.00980409 0.018728" quat="0.00431726 0.708285 -4.18922e-06 0.705913" mass="0.580973" diaginertia="0.00108466 0.00106374 0.000212638"/>
+                    <joint name="arm_joint_5" pos="0 0 0" axis="0 0 1" limited="true" range="-2.53 2.53"/>
+                    <geom type="mesh" contype="0" conaffinity="0" group="1" density="0" rgba="0.803 0.824 0.82 1" mesh="upper_wrist_link"/>
+                    <geom type="mesh" rgba="0.803 0.824 0.82 1" mesh="upper_wrist_link"/>
+                    <body name="arm_end_effector_link" pos="-0.105 0 0.0285" quat="0.707105 0 -0.707108 0">
+                      <inertial pos="0 0 0" mass="0.0001" diaginertia="0.0001 0.0001 0.0001"/>
+                      <joint name="arm_joint_6" pos="0 0 0" axis="0 0 1" limited="true" range="-2.09 2.09"/>
+                    </body>
+                  </body>
+                </body>
+              </body>
+            </body>
+          </body>
+        </body>
+      </body>
+    </body>
+  </worldbody>
+   <actuator>
+    <velocity joint="omni_joint_x" ctrlrange="-2.0 2.0" kv='100' ctrllimited="true" />
+    <velocity joint="omni_joint_y" ctrlrange="-2.0 2.0" kv='100' ctrllimited="true" />
+    <velocity joint="omni_joint_theta" ctrlrange="-2.0 2.0" kv='90' ctrllimited="true" />
+    <velocity joint="arm_joint_1" ctrlrange="-2.0 2.0" kv='100' ctrllimited="true" />
+    <velocity joint="arm_joint_2" ctrlrange="-2.0 2.0" kv='100' ctrllimited="true" />
+    <velocity joint="arm_joint_3" ctrlrange="-2.0 2.0" kv='90' ctrllimited="true" />
+    <velocity joint="arm_joint_4" ctrlrange="-2.0 2.0" kv='80' ctrllimited="true" />
+    <velocity joint="arm_joint_5" ctrlrange="-2.0 2.0" kv='70' ctrllimited="true" />
+    <velocity joint="arm_joint_6" ctrlrange="-2.0 2.0" kv='50' ctrllimited="true" />
+  </actuator>
```

Comment:
I autogenerated the xml files based on urdf files and copied these random controllers from the `kinova.xml` file. Is it alright or should I remove these controllers?

Reply:
These controllers are fine. Tuning is up to the user ;)